src/comodo/jaxsimSimulator/jaxsimSimulator.py

```diff
@@ -0,0 +1,465 @@
+from __future__ import annotations
+
+import logging
+import math
+import pathlib
+from enum import Enum
+from typing import Any, Optional, Tuple
+
+import jax.numpy as jnp
+import jaxsim
+import jaxsim.api as js
+import numpy as np
+import numpy.typing as npt
+from jaxsim import VelRepr, integrators
+from jaxsim.mujoco import MujocoVideoRecorder
+from jaxsim.mujoco.loaders import UrdfToMjcf
+from jaxsim.mujoco.model import MujocoModelHelper
+from jaxsim.mujoco.visualizer import MujocoVisualizer
+from jaxsim.rbda.contacts.common import ContactsParams
+from jaxsim.rbda.contacts.soft import SoftContacts
+from jaxsim.rbda.contacts.relaxed_rigid import (
+    RelaxedRigidContacts,
+    RelaxedRigidContactsParams,
+)
+from jaxsim.rbda.contacts.rigid import RigidContacts, RigidContactsParams
+from jaxsim.rbda.contacts.visco_elastic import ViscoElasticContacts
```

Comment:
You can import all these from `jaxsim.rbda.contacts`.

Reply:
Done in 2dcea841be59e5a81bc13e8d6bc6485ed322c68a